src/logging.cc

```diff
@@ -161,6 +161,8 @@ static const char* DefaultLogDir() {
   return "";
 }
 
+GLOG_DEFINE_int32(logfile_mode, 0644, "Log file mode/permissions");
```

Comment:
I'd still like the opinion of @shinh or @ukai on the default mode (I'm really just a "helper" maintainer), but this seems good to me.

My personal preference for flag description string is "not capitalized, no period", because they're in parentheses within an existing "sentence", like this:

```
    -stop_logging_if_full_disk (Stop attempting to log to disk if the disk is
      full.) type: bool default: false
```

I only looked at the flags immediately surrounding it, but since it's already kind of random, I'll have to live with this (for now), thanks! :wink: